app/Instructor/attendance/Config.php

```diff
@@ -0,0 +1,22 @@
+<?php
```

Comment:
@daxpatel1 you need to remove this file from Github. You should instead use `include('../../config.php');`